src/ArrayOfArraysView.hpp

```diff
@@ -558,6 +579,12 @@ class ArrayOfArraysView
     m_offsets.move( space, touch );
   }
 
+
+  void moveValues( MemorySpace const space, bool touch=true ) const
```

Comment:
What did you need this for that `move` didn't accomplish?

Reply:
Oh....It was to just move the "values" and not any of the other stuff. I think it is unnecessary now. I will probably remove this.